src/SprykerEco/Zed/Unzer/Business/Payment/Mapper/UnzerPaymentMapper.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Business\Payment\Mapper;
+
+use Generated\Shared\Transfer\PaymentUnzerTransactionTransfer;
+use Generated\Shared\Transfer\PaymentUnzerTransfer;
+use Generated\Shared\Transfer\UnzerBasketTransfer;
+use Generated\Shared\Transfer\UnzerCustomerTransfer;
+use Generated\Shared\Transfer\UnzerPaymentResourceTransfer;
+use Generated\Shared\Transfer\UnzerPaymentTransfer;
+use Generated\Shared\Transfer\UnzerTransactionTransfer;
+
+class UnzerPaymentMapper implements UnzerPaymentMapperInterface
+{
+    /**
```

Comment:
Imho it would make sense to explain what's the constant is for. What you think about that?